eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -293,13 +322,21 @@ FakeMaven withTojoAttribute(final ForeignTojos.Attribute attribute, final Object
     }
 
     /**
-     * Path to 'eo-foreign.csv' or 'eo-foreign.json' file after all changes.
+     * Path to eo-foreign.* file after all changes.
      * @return Path to eo-foreign.* file.
      */
     Path foreignPath() {
         return this.workspace.absolute(Paths.get("eo-foreign.csv"));
     }
 
+    /**
+     * Path to or eo-external.* file after all changes.
+     * @return Path to eo-foreign.* file.
+     */
+    Path externalPath() {
```

Comment:
@maxonfjvipon Looks like you are using that method only inside `FakeMaven` class. So, I believe we might put the `private` modifier here.

Reply:
@volodya-lombrozo it will be used outside of the `FakeMaven` in the future PRs (just like `foreignPath`) so I think there's no need to make it private now to remake later

Reply:
@maxonfjvipon I believe each new PR has to leave the entire codebase in a final and consistent state. So, we must have either a puzzle or a complete code. Otherwise, we risk losing your knowledge about "future" plans.